### PR TITLE
Tapping on User Statistics to navigate to MY TRACKS . 

### PR DESCRIPTION
**Pull request by asaikarthikeya**
#553 
It is natural tendency to click on the statistics to view complete tracking details rather than viewing it explicitly. 

https://user-images.githubusercontent.com/70392921/103418398-77bcf200-4bb4-11eb-8a59-4e30f0c39a72.mp4

### CODE REVIEW

**Comment by asaikarthikeya** [comment]
@dewall sir please review.

File: org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java [hunk 1]
```diff
@@ -18,18 +18,28 @@
...
 import com.squareup.otto.Bus;
 
 import org.envirocar.app.BaseApplication;
 import org.envirocar.app.BaseApplicationComponent;
+import org.envirocar.app.R;
 
 import javax.inject.Inject;
 
+import butterknife.BindView;
+
```
Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java:42]: not needed here

File: org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java [hunk 2]
```diff
@@ -41,6 +51,11 @@
     @Inject
     protected Bus mBus;
 
+    // BindView variables.
+    @BindView(R.id.navigation)
+    protected BottomNavigationView navigationBottomBar;
+
+
```
Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java:58]: undo all this changes

File: org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java [hunk 3]
```diff
@@ -73,4 +88,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    public void GotoMYTRACKS(View view) {
+        navigationBottomBar.setSelectedItemId(R.id.navigation_my_tracks);
+    }
```
Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java:93]: not to be done here.
Please follow the proper structure.
Do this on DashboardFragment 
for eg-: BottomNavigationView bottomView= getActivity.findViewById(R.id.navigation);
then use ButterKnife @Onlick on user statistics view id 
then you can select item

File: org.envirocar.app/src/org/envirocar/app/handler/userstatistics/UserStatisticsProcessor.java [hunk 4]
```diff
@@ -20,6 +20,8 @@
 
 import android.content.Context;
 import android.content.SharedPreferences;
+import android.util.Log;
+import android.view.View;
```
Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/handler/userstatistics/UserStatisticsProcessor.java:25]: undo

File: org.envirocar.app/src/org/envirocar/app/handler/userstatistics/UserStatisticsProcessor.java [hunk 5]
```diff
@@ -61,7 +63,7 @@
     private static final String PREF_KEY_TOTALDURATION = "totalDuration";
 
 
-    protected static final class UserStatisticsHolder {
+    protected static final class UserStatisticsHolder{
```
Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/handler/userstatistics/UserStatisticsProcessor.java:66]: same here, undo

**Review by arvindnegi1 — CHANGES_REQUESTED** [review]

**Comment by arvindnegi1** [comment]
@asaikarthikeya  Please read my comments  ;)
also remove the unwanted commit that is not associated with this PR.
Please change the base branch to develop.
And always make PRs to develop branch.

**Comment by asaikarthikeya** [comment]
@arvindnegi1 Sure . I will make the necessary changes and update you. thank you

**Pull request closed without merging**